content/audio.md

```diff
@@ -82,23 +134,13 @@ A device may be muted in the ALSA mixer, which will override any PulseAudio volu
 alsamixer
 ```
 
-You can navigate between the different volume meters using the left and right arrow keys. Each meter can be adjusted using the up and down arrow keys. An "MM" at the bottom of a meter indicates that meter is muted. If the PulseAudio Volume Control shows that sound is playing, but you don't hear any sound, try unmuting all of the volume meters in alsamixer by pressing the <kbd>M</kbd> key while each meter is selected.
+You can navigate between the different volume meters using the left and right arrow keys. Each meter can be adjusted using the up and down arrow keys. An "MM" at the bottom of a meter indicates that meter is muted. If the PulseAudio Volume Control shows that sound is playing, but you don't hear any sound, try unmuting all of the volume meters in alsamixer by pressing the M key while each meter is selected.
 
-![alsamixer](/images/audio/alsamixer-main.png)
+![audio_9](images/audio/audio_9.png)
 
 If your system has more than one sound card (for example, an Intel sound card and an NVIDIA sound card in switchable-graphics systems), you can switch between them by pressing F6.
 
-![alsamixer's sound card selection](/images/audio/alsamixer-soundcard.png)
-
-## Check if Hardware is Detected
-
-If ALSA doesn't list a sound card, it may not be physically detected by the system at all. If the Linux kernel sees a sound card, it will show up in your `lspci` output. This command will list every sound card your system detects, and show the driver being used for each one:
-
-```
-lspci -v | grep -A6 Audio
-```
-
-If the output from that command is blank, then your system isn't detecting any sound cards (which could indicate a hardware issue.)
+![audio_10](images/audio/audio_10.png)
 
 ## Other Useful Commands
```

Comment:
Let's take this opportunity to make the usage of `apt install --reinstall` vs `apt reinstall` consistent. They do the same thing, so I'd just go with `apt reinstall`.